### ISSUE TITLE
inline P-Header causes a wrong "bad tie"

### ISSUE THREAD

**Issue by bwl21** [moinejf/abc2svg#64]
```
X:3057
F:3057_wrong-bad-tie-message
T:wrong bad tie message
M:4/4
L:1/4
Q:1/4=120
I:transpose 2
K:D
%%score 1
V:1 clef=treble
c4|  E3 -
[P:Refr] E |]
```

throws a "noname.abc:11:6 Error: Bad tie"

**Comment by bwl21** [comment]
I investigated it on 1.15.3 and got "bad tie"
![screenshot_1057](https://user-images.githubusercontent.com/2022976/33543337-ee5078be-d8d6-11e7-8dc7-c26bac52402f.jpg)


using 1.14.4 the "bad tie" is gone. I see an "underfull line" which i think is ok.

![screenshot_1058](https://user-images.githubusercontent.com/2022976/33543357-024e5502-d8d7-11e7-89a0-47eb2e10f3c6.jpg)

@moinejf if you find this behavor reasonable, then we can close this issue. Maybe it was fixed by #62
